wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/UserService.java

```diff
@@ -83,6 +84,22 @@ public UserService(
     this.userSelfCreationAllowed = userSelfCreationAllowed;
   }
 
+  @GET
+  @Path("/id")
+  @Produces(TEXT_PLAIN)
+  @Operation(
+      summary = "Get current user's id",
+      responses = {
+        @ApiResponse(
+            responseCode = "200",
+            description =
+                "The response contains current user's id ('0000-00-0000' is returned for the anonymous user)"),
+      })
+  public String getId() {
+    return userId();
```

Comment:
Where the new request is going to be used?

Reply:
in the updated docs see the `Reviewers` section in the description. Basically, for now I do not think we have a better option for retrieving user id for various operations described in the docs